enocean/__init__.py

```diff
@@ -85,22 +86,47 @@
 CO_RD_NUMSECUREDEVICES = 0x29          # Read number of taught in secure devices
 CO_RD_SECUREDEVICE     = 0x30          # Read secure device by ID
 
+#List of Event Codes
+SA_RECLAIM_NOT_SUCCESSFUL  = 0x01      # Informs the backbone of a Smart Ack Client to not successful reclaim.
+SA_CONFIRM_LEARN           = 0x02      # Used for SMACK to confirm/discard learn in/out
+SA_LEARN_ACK               = 0x03      # Inform backbone about result of learn request
+CO_READY                   = 0x04      # Inform backbone about the readiness for operation
+CO_EVENT_SECUREDEVICES     = 0x05      # Informs about a secure device
+CO_DUTYCYCLE_LIMIT         = 0x06      # Informs about duty cycle limit
+CO_TRANSMIT_FAILED         = 0x07      # Informs that the device was not able to send a telegram.
+
+#Smart Acknowledge Defines:
+SA_WR_LEARNMODE        = 0x01          # Set/Reset Smart Ack learn mode
+SA_RD_LEARNMODE        = 0x02          # Get Smart Ack learn mode state
+SA_WR_LEARNCONFIRM     = 0x03          # Used for Smart Ack to add or delete a mailbox of a client
+SA_WR_CLIENTLEARNRQ    = 0x04          # Send Smart Ack Learn request (Client)
+SA_WR_RESET            = 0x05          # Send reset command to a Smart Ack client
+SA_RD_LEARNEDCLIENTS   = 0x06          # Get Smart Ack learned sensors / mailboxes
+SA_WR_RECLAIMS         = 0x07          # Set number of reclaim attempts
+SA_WR_POSTMASTER       = 0x08          # Activate/Deactivate Post master functionality
+
 SENT_RADIO_PACKET              = 0xFF
 SENT_ENCAPSULATED_RADIO_PACKET = 0xA6
 
 logger = logging.getLogger('EnOcean')
 
-class EnOcean():
 
+class EnOcean(SmartPlugin):
+    PLUGIN_VERSION = "1.1.2"
```

Comment:
The plugin version should start with the version of SH. So I suggest to use `1.3.2` or `1.3.0` instead.